src/app/services/navbar.service.ts

```diff
@@ -47,11 +49,24 @@ export class NavbarService {
       this.activeItem = this._items[0];
     } else if (this.isAnalysis) {
       this.activeItem = this._items[1];
+      this.configFractalis();
     } else if (this.isExport) {
       this.activeItem = this._items[2];
     }
   }
 
+  private configFractalis() {
+    if (this.firstTimeAnalysisTabActivated) {
```

Comment:
Instead of storing this here, you could ask the fractalis service if it was already setup.